nodejs/nodejs-cloud-run-hello-world/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "nodejs-cloud-run-hello-world",
+  "version": "0.0.1",
+  "description": "Hello World application for Cloud Run",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js",
+    "debug": "node --inspect=9229 index.js",
+    "lint": "eslint **.js",
+    "fix": "eslint --fix *.js",
+    "test": "mocha test/*.test.js --check-leaks --timeout 5000"
```

Comment:
If you replaced this with:

```
c8 mocha test/*.test.js --check-leaks --timeout 5000
```

and `npm i c8 --save-dev` you'll get test coverage for free, your call on whether you want to do so :smile: